rotkehlchen/tests/unit/decoders/accounting_settings/test_default_settings.py

```diff
@@ -0,0 +1,338 @@
+EXAMPLE_ADDRESS = string_to_evm_address('0x4bBa290826C253BD854121346c370a9886d1bC26')
+
+# Some utility timestamps. They don't matter here and are used only for mocked prices.
+TIMESTAMP_1_SECS = Timestamp(1624395186)
+TIMESTAMP_1_MS = ts_sec_to_ms(TIMESTAMP_1_SECS)
+TIMESTAMP_2_SECS = Timestamp(1628994441)
+TIMESTAMP_2_MS = ts_sec_to_ms(TIMESTAMP_2_SECS)
+
+
+@pytest.fixture(name='airdrops_taxable')
+def fixture_airdrops_taxable():
+    return True
+
+
+@pytest.fixture(name='gas_taxable')
+def fixture_gas_taxable():
+    return True
+
+
+@pytest.fixture(name='accounting_pot')
+def fixture_accounting_pot(accountant, airdrops_taxable, gas_taxable):
+    pot: 'AccountingPot' = accountant.pots[0]
+    with pot.database.user_write() as write_cursor:
+        taxable_ledger_actions = [LedgerActionType.AIRDROP] if airdrops_taxable else []
+        pot.database.set_settings(
+            write_cursor=write_cursor,
+            settings=ModifiableDBSettings(
+                taxable_ledger_actions=taxable_ledger_actions,
+                include_gas_costs=gas_taxable,
+            ),
+        )
+    with pot.database.conn.read_ctx() as cursor:
+        settings = pot.database.get_settings(cursor)
+    pot.reset(
+        settings=settings,
+        start_ts=Timestamp(0),  # timestamps don't matter since they are used only for querying
+        end_ts=Timestamp(0),  # histroy, which does not happen in the tests in this file
+        report_id=1,
+    )
+    return pot
+
+
+def _gain_one_ether(
+        transactions: 'TransactionsAccountant',
+        event_type: 'HistoryEventType' = HistoryEventType.RECEIVE,
+        event_subtype: 'HistoryEventSubType' = HistoryEventSubType.NONE,
+) -> None:
+    """Helper function to gain 1 ETH, so that spending events have what to spend"""
+    eth_gain_event = HistoryBaseEntry(
+        event_identifier=EXAMPLE_EVM_HASH,
+        sequence_index=0,
+        timestamp=TIMESTAMP_1_MS,
+        location=Location.BLOCKCHAIN,
+        location_label=EXAMPLE_ADDRESS,
+        asset=A_ETH,
+        balance=Balance(amount=ONE),
+        notes='Received 1 ETH',
+        event_type=event_type,
+        event_subtype=event_subtype,
+    )
+    consumed_num = transactions.process(
+        event=eth_gain_event,
+        events_iterator=iter([]),
+    )
+    assert consumed_num == 1
+
+
+def test_accounting_no_settings(accounting_pot: 'AccountingPot'):
+    """Test that if there are no settings provided, the event is not taken into account"""
+    event = HistoryBaseEntry(
+        event_identifier=EXAMPLE_EVM_HASH,
+        sequence_index=0,
+        timestamp=TimestampMS(0),
+        location=Location.BLOCKCHAIN,
+        event_type=HistoryEventType.INFORMATIONAL,
+        event_subtype=HistoryEventSubType.NONE,
+        asset=A_ETH,
+        balance=Balance(),
+    )
+    was_skipped = False
+
+    def log_mock(text):
+        assert 'Skipping...' in text
+        nonlocal was_skipped
+        was_skipped = True
```

Comment:
Don't do this. Do not rely on a log entry like this in a test. Check what is not there. So no processed events, no pnl change etc.